python/test/test_state.py

```diff
@@ -69,6 +69,10 @@ def test_timestamp(self):
         self.assertTrue(state.is_deprecated(0.1))
         state.reset_timestamp()
         self.assertFalse(state.is_deprecated(0.1))
+        time.sleep(0.2)
+        self.assertTrue(state.is_deprecated(0.1))
+        state.set_timestamp(time.localtime())
```

Comment:
Apart from that change here, you need to `import datetime` on line 1.
```suggestion
        state.set_timestamp(datetime.datetime.now().timestamp())
```
Additionally, please add `'set_timestamp'` on line 13 (i cant make suggestions for those lines)